frontend/packages/policy-editor/src/data-mocks/policyActionMocks.ts

```diff
@@ -1,21 +1,25 @@
 import { PolicyAction } from '../types';
 
-export const mockActionTitle1: string = 'Action 1';
-export const mockActionTitle2: string = 'Action 2';
-export const mockActionTitle3: string = 'Action 3';
+export const mockActionId1: string = 'a1';
```

Comment:
We should add one more test case where the actionId does not exist in the translation file. 😊